recipes/mpfr/all/test_v1_package/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.3.0)
+project(test_package C)
+
+include("${CMAKE_BINARY_DIR}/conanbuildinfo.cmake")
+conan_basic_setup(TARGETS)
+
+add_executable(${PROJECT_NAME} ../test_package/test_package.c)
+target_link_libraries(${PROJECT_NAME} CONAN_PKG::mpfr)
```

Comment:
```suggestion
cmake_minimum_required(VERSION 3.1)
project(test_package)

include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
conan_basic_setup(TARGETS)

add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/../test_package
                 ${CMAKE_CURRENT_BINARY_DIR}/test_package)
```

Reply:
Hi @SpaceIm,
Unless I am missing something, this approach does not work because it causes the CMakeList.txt file in ../test_package to be parsed as part of the test_v1_package which leads to all sorts of problems. I spent several hours trying to make this work in another recipe where the same suggestion was made., Is there a working example of this strategy somewhere that I can study?

Reply:
This pattern works well for easy project (fmt zlib etc) but this is not one of those so it's okay if it does not get applied here :)